packages/data-point/README.md

```diff
@@ -3324,6 +3325,29 @@ Example at: [examples/extend-entity-keys.js](examples/extend-entity-keys.js)
 
 Example at: [examples/extend-entity-reusability.js](examples/extend-entity-reusability.js)
 
+### <a name="global-entity-options">Globally setting options for entities</a>
+You may set global params for entry types, allowing you to set these options once and have them be consistent. We do this using `entityOverrides` within the options object.
+
+**EXAMPLE**
+
+```js
+const options = {
+  entityOverrides: {
+    'request': {
+      params: {
+        inspect: true
+      }
+    }
+  }
+}
+
+dataPoint.resolve('request:xyz', {}, options)
+```
+The preceeding example will make it so every `'request'` entity will have the inpsect param set to true. This can be done with any entity and its respectful parameters.
```

Comment:
seeing a few typos here (preceeding, inpsect); also please change "respectful" to "respective"